tests/cases/fourslash/server/renameNamespaceImport.ts

```diff
@@ -0,0 +1,22 @@
+/// <reference path='../fourslash.ts' />
+
+// @Filename: /lib/tsconfig.json
+//// {}
+
+// @Filename: /lib/index.ts
+//// const unrelatedLocalVariable = 123;
+//// export const someExportedVariable = unrelatedLocalVariable;
+
+// @Filename: /src/tsconfig.json
+//// {}
+
+// @Filename: /src/index.ts
+//// import * as /*i*/lib from '../lib/index';
```

Comment:
I’m surprised this is doing the multi-project thing without `references` in `src/tsconfig.jon`. You’re sure this test is doing what you think it is?

Reply:
I think so, yes. If I revert my change, it fails in exact the same way as the bug does, and debugging it, it seems the same code path is executed. Why are you surprised it works? I'm not sure I understand.

Reply:
My mental model of this test is that `/lib/tsconfig.json` would never come into play because `/src/tsconfig.json` directly includes all source files; in other words, you could delete `/lib/tsconfig.json` and the shape of the `/src/tsconfig.json` project would be unaffected. @amcasey can probably explain my mistake in this thinking.

Reply:
We look for the "default" tsconfig, even if we know the file is in a configured project.